docs/sources/auth/generic-oauth.md

```diff
@@ -42,6 +43,11 @@ tls_client_ca =
 
 Set `api_url` to the resource that returns [OpenID UserInfo](https://connect2id.com/products/server/docs/api/userinfo) compatible information.
 
+Grafana will always generate a unique `state` value, for security purposes. However, if you need to influence the content of the state parameter in the oAuth requests, this can be done through by setting `state`
+- if the value of `state` containes the text `<random>`, the text `<random>` will automatically be replaced with a generated random value
+- if the value of `state` does NOT contain the text `<random>`, an automatically generated random value will be appended to the provided `state` value
+- if no value is provided for `state`, it will be set to an automatically generated random value
+
```

Comment:
```suggestion
Grafana generates a unique `state` value, for security purposes. You can also modify the value of the state parameter.
- if `state` includes the text `<random>`, the text `<random>` is replaced with a randomly generated value.
- if `state` does not include the text `<random>`, an automatically generated value is appended to the `state` value.
- if no value is provided, `state` is set to automatically generated random value.

```

Reply:
done

Reply:
Please check this doc with a local build. I had an issue the other day where text enclosed in pointy braces did not appear when the page was generated in Hugo.